src/utils/thread_helper.cpp

```diff
@@ -920,6 +920,43 @@ void ThreadHelper::DetachGracefullyCallback(void)
 }
 
 #if OTBR_ENABLE_TELEMETRY_DATA_API
+#if OTBR_ENABLE_BORDER_ROUTING
+void ThreadHelper::GetNodeDivergenceInfo(threadnetwork::TelemetryData_NodeDivergenceInfo *aNodeDivergenceInfo)
+{
+    bool      isExternalRoutePublished = false;
+    bool      isDefaultRoutePublished  = false;
+    Ip6Prefix prefix;
+    uint16_t  rloc16 = otThreadGetRloc16(mInstance);
+
+    otNetworkDataIterator iterator = OT_NETWORK_DATA_ITERATOR_INIT;
+    otExternalRouteConfig config;
+
+    // Iterate leader network data
+    while (otNetDataGetNextRoute(mInstance, &iterator, &config) == OT_ERROR_NONE)
+    {
+        // filter out the config that is not stable or unmatched rloc16
```

Comment:
Suggest removing this comment here and ones below as they simply repeat what the code does and the logic is clear from the code.

This link has some useful advice on this: [code-health-to-comment-or-not-to-comment](https://testing.googleblog.com/2017/07/code-health-to-comment-or-not-to-comment.html) (or Google internal link [go/tott/563](https://goto.google.com/tott/563))

Reply:
Thanks! The link is helpful! And I removed the extra comment line.